src/app/Home.module.css

```diff
@@ -7,7 +7,7 @@
     margin-right: 1em;

```

Comment:
[Please Change]: Above this, change `.landingTitle` to `.landingBackground h1` to match the others - this will fix the small title text (remove the color as well).

Reply:
Done